types/time.go

```diff
@@ -2089,6 +2090,26 @@ func (t Time) convertDateFormat(b rune, buf *bytes.Buffer) error {
 	return nil
 }
 
+func formatIntWidth2(i int) string {
+	if i < 10 {
+		return "0" + strconv.FormatInt(int64(i), 10)
+	}
+	return strconv.FormatInt(int64(i), 10)
+}
+
+func formatIntWidth4(i int) string {
```

Comment:
Could you implement a `formatIntWidth` function, and pass the width as a parameter?